include/PreCharge/dev/MAX22530.hpp

```diff
@@ -0,0 +1,34 @@
+#include <EVT/io/SPI.hpp>
+#include <PreCharge/PreCharge.hpp>
+#include <cstdint>
+
+namespace PreCharge {
+
+/**
+ * Handles reading the voltage values & converting it to decivolts from the
```

Comment:
NIT: Goes into a bit too much detail. Doesn't need to go beyond "Handles reading voltage data from the MAX22530 ADC